dll/win32/iernonce/registry.cpp

```diff
@@ -5,15 +5,9 @@
  * COPYRIGHT:   Copyright 2021 He Yang <1160386205@qq.com>
  */
 
+#include "iernonce.h"
 
-#include <cassert>
-#include <cstdlib>
-#include <shlwapi.h>
-#include <windows.h>
-#include <winreg.h>
-
-#include "registry.h"
-#include "dialog.h"
+extern RUNONCEEX_CALLBACK g_Callback;
```

Comment:
Might be moved into the pch header too.

Reply:
actually only `registry.cpp` will need to use this `g_Callback`, is this still needed?

Reply:
ok then keep it there.